lib/cn/kvset.c

```diff
@@ -1740,6 +1740,12 @@ kvset_lookup(
     return kvset_lookup_val(ks, &vref, vbuf);
 }
 
+u64
+kvset_get_nodeid(struct kvset *ks)
```

Comment:
const kvset

Reply:
copy-n-paste, grrr..  I would go fix them all but we should do that in separate ticket...
Done.